components/datetime/benches/fixtures/tests/components.json

```diff
@@ -22,7 +22,7 @@
                 "locale": "en",
                 "options": {
                     "components": {
-                        "year": "numeric",
+                        "year": "Numeric",
                         "month": "numeric",
                         "day": "numeric"
```

Comment:
> thought

I would prefer to see uniform casing in the JSON data, rather than a mixture of`"Numeric"` and `"numeric"`, but we can fix this. See the suggestion comment where the `Year` enum is defined.

Reply:
Done